### PR TITLE
Test writing

### PR DESCRIPTION
**Pull request by toadslop**

### CODE REVIEW

File: test/factories/list.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FactoryBot.define do
```
Comment (Naokimi) [inline test/factories/list.rb:1]: the factories would usually be plural, so test/factories/lists.rb

File: test/factories/region.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :region do
+    sequence(:name) { |n| "Region #{n}" }
+  end
+end
```
Comment (Naokimi) [inline test/factories/region.rb:5]: missing newline

File: test/fixtures/list_mountains.yml [hunk 3]
```diff
@@ -1,15 +1,5 @@
...
   number: 1
-  list_id: 1
-  mountain_id: 1
 
-two:
-  number: 2
-  list_id: 1
-  mountain_id: 1
+second:
+  number: 2
```
Comment (Naokimi) [inline test/fixtures/list_mountains.yml:5]: missing newline

File: test/fixtures/mountains.yml [hunk 4]
```diff
@@ -4,16 +4,16 @@ Fuji:
...
   altitude: 1111
   terrain_diff: ★★
   physical_diff: ★★
   length: '一泊二日'
-  region_id: 2
   lat: 35.360747509167
-  lng: 138.72735534793
\ No newline at end of file
+  lng: 138.72735534793
+  region_id: 1
```
Comment (Naokimi) [inline test/fixtures/mountains.yml:19]: missing newline

Comment (toadslop) [reply test/fixtures/mountains.yml:19]: I don't understand where the newline should be or why.

File: test/models/mountain_test.rb [hunk 5]
```diff
@@ -35,6 +35,18 @@ def setup
     assert @mountain.lists.count >= 2
   end
 
+  test 'should not save a duplicate mountain' do
+    duplicate_mountain = Mountain.new(
```
Comment (Naokimi) [inline test/models/mountain_test.rb:39]: it doesn't save if the duplicated mountain is exactly the same
what happens if only one or a few of the variables are the same? will it still save?

**Pull request merged**